crates/ra_lsp_server/src/main_loop/handlers.rs

```diff
@@ -758,7 +758,7 @@ pub fn handle_code_lens(
     // Gather runnables
     for runnable in world.analysis().runnables(file_id)? {
         let title = match &runnable.kind {
-            RunnableKind::Test { .. } | RunnableKind::TestMod { .. } => "▶️Run Test",
+            RunnableKind::Test { .. } | RunnableKind::TestMod { .. } => "▶️&#xFE0E;Run Test",
```

Comment:
I don't think this will work correctly in non html based editors. Could you please use `\u{fe0e}`. (or at least I think that is the correct rust unicode codepoint syntax)

Reply:
oh you right